src/main/java/org/jabref/logic/texparser/DefaultTexParser.java

```diff
@@ -121,13 +121,18 @@ private void matchNestedFile(Path file, List<Path> texFiles, List<Path> referenc
         while (includeMatch.find()) {
             String include = includeMatch.group(INCLUDE_GROUP);
 
-            Path inputFile = file.getParent().resolve(
+            Path nestedFile = file.getParent().resolve(
                     include.endsWith(TEX_EXT)
                             ? include
                             : String.format("%s%s", include, TEX_EXT));
 
-            if (!texFiles.contains(inputFile)) {
-                referencedFiles.add(inputFile);
+            if (!nestedFile.toFile().exists()) {
+                LOGGER.error(String.format("Nested file does not exist: %s", nestedFile));
```

Comment:
This might be controversial, but I'd argue that you shouldn't, add this to the logs, as in my opinion this is not a relevant message for the user.

Reply:
Yep, I was hesitating about it. But `DefaultTexParser` also logs files that do not exist in the `parse()` method, so finally I have added the message. Do you think we should remove only this message or both?

https://github.com/JabRef/jabref/blob/034227b82b51010cd6bb8d0573dbcdbe91a95cfc/src/main/java/org/jabref/logic/texparser/DefaultTexParser.java#L73-L76

Reply:
I think that you can keep that message, although it can only really occur if the file was deleted after parsing.

Reply:
Yes, that is the only case, if I am right. It is something like defensive programming. Maybe in the future someone will use the `parse()` method with a list of wrong files.

Good. So I am going to remove only the message in `matchNestedFile()`.